styles/pages/contact/contact.scss

```diff
@@ -7,11 +7,9 @@
 
 .contact__wrapper {
```

Comment:
consider creating a wrapper utility class to extend with standardized margins, padding, etc. for most pages